### PR TITLE
Dropdown filter

### PR DESCRIPTION
**Pull request by janicelam21**
Revised dropdown button now sorts reviews based on selected filter
@dlwong @poema-cavalcanti @creativespoon

### CODE REVIEW

File: client/src/components/App.jsx [hunk 1]
```diff
@@ -23,18 +24,32 @@ class App extends React.Component {
...
       })
       .catch((error) => {
-        // eslint-disable-next-line no-console
         console.log(error);
       });
   }
 
+  handleSortedReviews(sort) {
+    console.log(sort)
+    axios.get('/80/reviews', {
```
Comment (poema-cavalcanti) [inline client/src/components/App.jsx]: Why 80? Is there a way to not have the restaurant_id hardcoded in? Maybe have it passed into the handleSortedReviews function as an argument?

**Comment by janicelam21** [comment]
I modified the restaurants_id to be randomized instead of hardcoded. 
The tests will be a separate PR.

File: client/src/components/App.jsx [hunk 2]
```diff
@@ -10,31 +10,47 @@ class App extends React.Component {
     this.state = {
       reviews: [],
     };
+    this.randRestID = '',
```
Comment (poema-cavalcanti) [inline client/src/components/App.jsx]: Might make sense to make this part of the state instead for organization.

**Comment by janicelam21** [comment]
moved restaurantID to be in state

**Pull request merged**